installation/install.sh

```
@@ -3592,13 +3633,23 @@ get_zstack_repo(){
         fi
         echo $ZSTACK_YUM_REPOS |grep "ali" >/dev/null 2>&1
         if [ $? -eq 0 ]; then
-            ZSTACK_YUM_REPOS=$MIRROR_ALI_YUM_REPOS
-            ZSTACK_PROPERTIES_REPO=$MIRROR_ALI_YUM_REPOS
+            if [ x"$OS" = x"ALINUX4" ]; then
```

Shellcheck (0.11.0)

[style] Avoid x-prefix in comparisons as it no longer serves a purpose.

(SC2268)